pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -399,6 +399,15 @@ def test_corrwith_spearman(self):
         expected = Series(np.ones(len(result)))
         tm.assert_series_equal(result, expected)
 
+    @td.skip_if_no_scipy
+    def test_corrwith_spearman_with_tied_data(self):
+        # GH#48826
+        df = DataFrame({"A": [2, np.nan, 8, 9], "B": [0, 1, 1, 0]})
+        s = Series([0, 1, 1, 0])
```

Comment:
`s` -> `ser`

Also could you test with this as `bool` data like in the original issue?